tests/dummy/app/controllers/nodes/detail/index.js

```diff
@@ -19,6 +19,21 @@ export default Ember.Controller.extend({
             var bibliographic = target.checked;
             var contributorId = target.value;
             this.editedBibliographic[contributorId] = bibliographic;
+        },
+        addATag(tag) {
```

Comment:
Since we're doing this in two places, would it make sense to create a mixin to generalize this behavior? This could be a `TaggableMixin` that can be mixed into controllers and routes.